src/main/webapp/js/feedbackResponseComments.es6

```diff
@@ -347,25 +347,28 @@ function registerResponseCommentsEvent() {
 
     const clickHandlerMap = new Map();
     clickHandlerMap.set('.show-frc-add-form',
-            [showResponseCommentAddForm, ['recipientindex', 'giverindex', 'qnindex', 'sectionindex']]);
+        [showResponseCommentAddForm, ['recipientindex', 'giverindex', 'qnindex', 'sectionindex']]);
```

Comment:
setting it at 12 spaces from the start doesn't pass `npm run lint` test.
` Expected indentation of 8 spaces but found 12  indent`
Is the below indentation fine?
![final](https://user-images.githubusercontent.com/17481466/27073616-f2021fea-5013-11e7-8271-e0a86e3ed202.PNG)